lib/stat_tracker.rb

```diff
@@ -124,22 +45,30 @@ def average_goals_per_game
 		(GameModule.total_score(@games).sum.to_f / game_count).ceil(2)
 	end
 
-	def average_goals_per_season
-		season_goals_avg = {}
-		@games.each do |game|
-			season = game.season
-			if season_goals_avg[season] == nil
-				season_goals_avg[season] = [game.away_goals + game.home_goals]
-			else
-				season_goals_avg[season] << game.away_goals + game.home_goals
-			end
-		end
+	def average_goals_by_season
+		season_goals_avg = GameModule.season_goals(@games)
 		season_goals_avg.each do |season, goals|
 			season_goals_avg[season] = (goals.sum.to_f / goals.count.to_f).ceil(2)
 		end
 		return season_goals_avg
 	end
 
+	# def average_goals_by_season
```

Comment:
Is it alright to delete this commented out code when we merge it to main, or do you want to save this block of code for now.

Reply:
It's okay to delete.